src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -211,11 +211,17 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
           display: 'Sample Date',
           sortable: true,
           formatter: function(v) {
-            let val = '';
-            if (v) {
-              val = $filter('date')(new Date(v), 'dd-MMM-yyyy');
+            let dateResult = '';
+            const dateVal = v && v.split('-').map(val => Number(val));
```

Comment:
Can you use `&&` like this? OK if so, but harder to read for me at least

Reply:
Change back to if